indico/htdocs/js/indico/jquery/ajaxdialog.js

```diff
@@ -55,6 +55,7 @@
             closeButton: undefined, // include a close button at the bottom of the dialog. the inner text of the button
                                     // is configurable
             url: null, // url to get the form/dialog from
+            contentSelector: null, // selector to get the contents of the dialog (used instead of the url option)
```

Comment:
this is not a selector, your code expects an element. maybe it'd be better to use `content` instead and expect the calling code to do the `$('...').html()`?